samples/Samples.Schemas.Chat/Services/Chat.cs

```diff
@@ -20,7 +20,7 @@ public IEnumerable<Message> GetAllMessages()
     public IEnumerable<Message> GetMessageFromUser(string from)
     {
         lock (_messages)
-            return _messages.Where(x => string.Equals(x.From, from, StringComparison.InvariantCultureIgnoreCase)).ToList();
+            return _messages.Where(x => string.Equals(x.From, from, StringComparison.Ordinal)).ToList();
```

Comment:
This change should be reverted. It was explicitly set to a case insensitive search, as would be appropriate in this example.